src/utils/clientToSVGTranslator.ts

```diff
@@ -3,83 +3,35 @@
  * @license MIT
  */
 
-import { Point } from "../core/interfaces";
-
 import * as DOM from "./domUtils";
+import { PlottableSVGElement } from "./plottableElement";
+import { Translator } from "./translator";
+
+import { IComponent } from "../components";
 
 export class ClientToSVGTranslator {
   private static _TRANSLATOR_KEY = "__Plottable_ClientToSVGTranslator";
-  private _svg: SVGElement;
-  private _measureRect: SVGElement;
 
   /**
    * Returns the ClientToSVGTranslator for the <svg> containing elem.
    * If one already exists on that <svg>, it will be returned; otherwise, a new one will be created.
    */
-  public static getTranslator(elem: SVGElement): ClientToSVGTranslator {
-    let svg = DOM.boundingSVG(elem);
+  public static getTranslator(component: IComponent<any>): Translator {
+    let svg = DOM.boundingSVG(component.element().node() as SVGElement);
```

Comment:
shouldn't we make the component be `IComponent<SVGElement>` or somehting then? Or am I misunderstanding the type parameter